app/src/components/features/rules/ImportRulesModal/index.js

```diff
@@ -287,64 +295,41 @@ const ImportRulesModal = (props) => {
         />
       ) : null}
 
-      <Modal
-        style={{ display: isImportFromCharlesModalOpen ? "none" : "block" }}
-        className="modal-dialog-centered"
+      <RQModal
         open={isOpen}
         onCancel={toggleModal}
-        footer={null}
-        title={
-          <Row align="middle">
-            {isCharlesImportFeatureFlagOn && !showImportOptions && (
-              <Button
-                size="small"
-                style={{ marginRight: "6px" }}
-                onClick={() => setShowImportOptions(true)}
-                icon={<LeftOutlined style={{ fontSize: "14px" }} />}
-              />
-            )}
-
+        style={{ display: isImportFromCharlesModalOpen ? "none" : "block" }}
```

Comment:
again here. Shouldn't this be done with `className` ?

Reply:
yep, updating

Reply:
we can use classes but again we have to put `!important` for specificity